packages/ibm-products/src/components/CreateModal/CreateModal.tsx

```diff
@@ -90,7 +90,7 @@ export interface CreateModalProps
   /**
    * Specifies which DOM element in the form should be focused.
    */
-  selectorPrimaryFocus: ReactNode;
+  selectorPrimaryFocus: string;
```

Comment:
Same as above, why the shift to string?

Reply:
Because in `ComposedModal`, the `selectorPrimaryFocus` property type is `string`.
[ComposedModal.tsx#L207](https://github.com/carbon-design-system/carbon/blob/72fb7c06809d0039807f09be07419a3fa7d2fbf9/packages/react/src/components/ComposedModal/ComposedModal.tsx#L207)